### ISSUE TITLE
pouchdb-adapter-leveldb does not compile with node 12

### ISSUE THREAD

**Issue by jeremad** [pouchdb/pouchdb#7746]
### Issue
I cannot build `pouchdb-adapter-leveldb` with node 12. I get the following compilation error:

```
/home/jeremad/.node-gyp/12.1.0/include/node/v8.h:4125:31: note:   no known conversion for argument 1 from ‘v8::Isolate*’ to ‘v8::Local<v8::Context>’
In file included from ../../nan/nan_new.h:189,
                 from ../../nan/nan.h:203,
                 from ../src/batch.cc:3:
../../nan/nan_implementation_12_inl.h: In static member function ‘static Nan::imp::FactoryBase<v8::StringObject>::return_t Nan::imp::Factory<v8::StringObject>::New(v8::Local<v8::String>)’:
../../nan/nan_implementation_12_inl.h:337:37: error: no matching function for call to ‘v8::StringObject::New(v8::Local<v8::String>&)’
```

Bumping level to the latest version probably fixes the issue.

### Info
- Environment: Ubuntu 19.04, node 12.1, yarn 1.15.2
- Adapter: LevelDB

### Reproduce

in a package.json:
```
{ 
  "dependencies": { 
    "pouchdb-adapter-leveldb": "^7.0.0" 
  } 
} 
```
and just run `yarn` or `npm install`

**Comment by taschana** [comment]
Had the same issue, if you are interested in a temporary fix until it is published to npm package repository, try here: #7754

**Comment by jeremad** [comment]
@taschana sounds great, thanks!
